packages/flutter/example/lib/live_list/main.dart

```diff
@@ -49,9 +49,9 @@ class _MyAppState extends State<MyApp> {
                 height: double.infinity,
                 alignment: Alignment.center,
                 child: initFailed == null
-                    ? Column(
+                    ? const Column(
```

Comment:
> Analyzing flutter...                                            
> 
>   error • The constructor being called isn't a const constructor • packages/flutter/example/lib/live_list/main.dart:52:23 • const_with_non_const
> 
> 1 issue found. (ran in 1[9](https://github.com/parse-community/Parse-SDK-Flutter/actions/runs/4980883651/jobs/8914366754?pr=884#step:6:10).0s)

Remove the `const` keyword. The const Columns widget was introduced in Flutter `3.10.0` and can not be used in flutter `3.3.x` or `3.7.x`

Reply:
I delete it, I get a warning

```dart
Use 'const' with the constructor to improve performance
```
Are you sure about this?

Reply:
Deleted